FrontEnd/src/UserAuthentication/AlternateSignUp.jsx

```diff
@@ -0,0 +1,166 @@
+import { useNavigate, Link } from "react-router-dom";
+import { useState, useEffect, useRef } from "react";
+import { registerUser } from "./Auth";
+import { CATEGORIES } from "../Categories";
+import "./EntryPage.css";
+
+function SignUpPage({ theme }) {
```

Comment:
Why is this called "alternate"?

Reply:
On smaller screens, the signUp page stays different (it doesn't slide over like it does in bigger screens because of space)

Reply:
Maybe would be useful to have a comment at the top of the component explaining this, otherwise naming is a bit confusing.